### ISSUE TITLE
Frequently stuck at login due to segfault in greetd

### ISSUE THREAD

**Issue by xvbcfj** [pop-os/cosmic-greeter#57]
```
Jun 07 20:45:10 pop-os kernel: greetd[30377]: segfault at 7ffc49729f98 ip 0000782fd5075048 sp 00007ffc49729f90 error 6 in libc.so.6[782fd5028000+195000] likely on CPU 6 (core 6, socket 0)
```

**Comment by jackpot51** [comment]
This is probably related to #12.

**Comment by wash2** [comment]
Disabling `eCryptfs Key/Mount Management` with `pam-auth-update` avoids this and seems to fix #12 for me too after logging in and out a bunch. I think we could probably remove it by default because we don't encrypt the home directory, right?

**Comment by mmstick** [comment]
Yes, it'd be good to disable it if it's causing problems for PAM
